samples/snippets/evaluate_processor_version_sample_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+import mock
```

Comment:
Don't use mocks. The samples should be run against the actual service. LMK if you need assistance with setting up resources.

Reply:
This test still uses mocks.

See here about integration tests:
https://googlecloudplatform.github.io/samples-style-guide/#coverage

Reply:
(Side note: please don't resolve unaddressed comments without an "ack" and a reason why.)

Reply:
Sorry, I thought I responded to this comment. I can set up resources in the test project, however the evaluate and train methods can take a long time to complete (multiple hours in some cases) and the tests have to run sequentially to avoid race conditions with the operations tests.
Is there a recommended practice for these expensive methods?